source/settings/core.rst

```diff
@@ -3348,8 +3273,7 @@ See :ref:`settings` for list of all setting groups.
    :values: @string
 
    The compile-time directory PKG_STATEDIR (typically /var/lib/dovecot)
-   is hard-coded as the location of things such as the ssl-parameters.dat
-   file and the replicator database. The PKG_STATEDIR value is taken as
+   is hard-coded as the location of state files. The PKG_STATEDIR value is taken as
```

Comment:
I'd prefer this would be formatted like the rest of the paragraph/content (i.e. 80 cols).